Kim Edits (#50)
mulesoft/docs-partner-manager: modules/ROOT/pages/_partials/endpoint-as2-receive.adoc

Required: No +

| AS2 URL (Automatically generated at time of deployment)

> | The HTTP(S) address at where you receive AS2 messages.

Reviewer: ```suggestion
| The HTTP or HTTPS address where you receive AS2 messages.
```
Also, do not end such a fragment with a period unless this is part of a list in which one or more descriptions is a complete sentence.

Author: Thanks.

The port is automatically generated when you deploy a message flow that uses this endpoint.

> You'll be able to see it by selecting the receiving endpoint from your deployed message flow.

Reviewer: ```suggestion
You can see it by selecting the receiving endpoint from your deployed message flow.
```

> You get the hostname from your administrator who created it upon setup.

Reviewer: What does this mean: do you have to ask the admin, or is it just obvious by viewing the port? Do we mean...:
```suggestion
The hostname displayed is created by your administrator at setup.
```

Reviewer: Lines 23-25
"If your partner sends an encrypted signature in the message, choose the appropriate algorithm for validating that signature:
